addons/website/static/src/systray_items/website_switcher.xml

```diff
@@ -13,10 +13,16 @@
         </t>
         <t t-foreach="getElements()" t-as="element" t-key="element_index">
             <DropdownItem
-                t-esc="element.name"
                 onSelected="element.callback"
-                class="element.class"
-            />
+                class="element.class">
+                <t t-if="element.noDomain">
+                    <span class="fa fa-warning me-2 text-warning"
+                          t-att-data-tooltip="noDomainTooltip"
```

Comment:
Do we really need a property on `this` for this tooltip? Can't we add it directly in the DOM here?